xbmc/network/ZeroconfBrowser.cpp

```diff
@@ -57,7 +57,6 @@ CZeroconfBrowser::CZeroconfBrowser():mp_crit_sec(new CCriticalSection),m_started
   AddServiceType("_smb._tcp.");
 #endif
   AddServiceType("_ftp._tcp.");
-  AddServiceType("_htsp._tcp.");
```

Comment:
@opdenkamp Just realised that this could have killed your avahi discovery feature for pvr.hts?